.github/workflows/e2e-pull-request.yml

```diff
@@ -47,13 +47,8 @@ jobs:
     strategy:
       fail-fast:     false
       matrix:
-        test_groups:   [ 'wcpay', 'subscriptions', 'upe', 'upeSplit' ] # [TODO] Unskip blocks tests after investigating constant failures.
+        test_groups:   [ 'wcpay', 'subscriptions' ] # [TODO] Unskip blocks tests after investigating constant failures.
         test_branches: [ 'merchant', 'shopper' ]
-        exclude:
-          - test_groups: 'upe'
-            test_branches: 'merchant'
-          - test_groups: 'upeSplit'
-            test_branches: 'merchant'
```

Comment:
@timur27 I agree that we don't need to have too many sub groups for E2E tests. Moving UPE tests to shopper makes sense as long as it doesn't add too much time addition to the test runtime.

What is the time difference you've noticed after moving the tests to shopper?

Reply:
@achyuthajoy thanks, I think it's not more than 110 seconds but it's hard to confirm as of today because tests are failing on `develop`. I will get back if after fixing `develop` this branch will be extended by more than ~2mins but in the meantime let's assume this is the change.

Reply:
Sound good to me.